gravitee-apim-portal-webui-next/src/scss/theme/theme.scss

```
@@ -0,0 +1,43 @@
+@use '@angular/material' as mat;
+
+$mat-primary: (
+  main: #6c59bd,
+  lighter: #d3cdeb,
+  darker: #4f3ea6,
+  200: #6c59bd,
+  contrast : (
+    main: #ffffff,
+    lighter: rgba(#000000, 0.87),
+    darker: #ffffff,
+  )
+);
+
+$mat-accent: (
+  main: #e8e3fc,
+  lighter: #f8f7fe,
+  darker: #ded8fb,
+  200: #e8e3fc,
+  contrast : (
+    main: #ffffff,
+    lighter: rgba(#000000, 0.87),
+    darker: #ffffff,
+  )
+);
```

In order to match the palette, change the main color to `e8def8` and darker to match container background in Figma. Also, all contrast to be dark since text is light.

```suggestion
$mat-accent: (
  main: #e8def8,
  lighter: #f8f7fe,
  darker: #e5deff,
  200: #e8def8,
  contrast : (
    main: #1d192b,
    lighter: #1d192b,
    darker: #1d192b,
  )
);
```

---

Done